qulice-checkstyle/src/test/resources/com/qulice/checkstyle/ChecksTest/NoJavadocForOverrideMethods/Valid.java

```diff
@@ -0,0 +1,27 @@
+/**
+ * This is not a real Java class. It won't be compiled ever. It is used
+ * only as a text resource in integration.ChecksIT.
+ */
+
+public final class Valid {
+
+    @Override
+    public void main() {
+    }
+
+    /**
+     * Javadoc.
+     */
+    public void note() {
+    }
+
+    @Override
```

Comment:
@HDouss can you also verify that normal comments like `//` or `/*` are still allowed above such methods?